tests/unit/spec/mozilla-smoothscroll.js

```diff
@@ -0,0 +1,76 @@
+/* For reference read the Jasmine and Sinon docs
+ * Jasmine docs: https://jasmine.github.io/2.3/introduction.html
+ * Sinon docs: http://sinonjs.org/docs/
+ */
+
+/* global describe, beforeEach, afterEach, it, expect, sinon, spyOn */
+
+describe('mozilla-smoothscroll.js', function () {
+
+    'use strict';
+
+    describe('Mozilla.smoothScroll scrolling', function () {
+
+        beforeEach(function () {
+            var scrollMarkup = [
+                '<div id="scrollMarkup">',
+                    '<nav id="nav">',
+                        '<a id="link1" href="#anchor1">Anchor 1</a>',
+                        '<a id="link2" href="#anchor2">Anchor 2</a>',
+                        '<a id="link3" href="#anchor3">Anchor 3</a>',
+                    '</nav>',
+                    '<div id="anchor1" style="height: 400px;"></div>',
+                    '<div id="anchor2" style="height: 400px;"></div>',
+                    '<div id="anchor3" style="height: 400px;"></div>',
+                '</div>'
+            ].join();
+
+            $(scrollMarkup).appendTo('body');
+        });
+
+        afterEach(function () {
+            $('#scrollMarkup').remove();
+        });
+
+        // TODO: This test works as long as Fx is specified as the testing browser,
```

Comment:
Instead of relying on a specific browser to run the tests, how about passing a stub variable used only for testing? I've used a similar approach elsewhere for read-only properties like `document.hidden`. All the tests really need to make sure if when the native API is not supported, jQuery gets called instead with the expected values etc, if that makes sense?